utils/colorTheme.js

```diff
@@ -3,5 +3,10 @@ export const newColorTheme = {
     100: '#005A86',
     200: '#DFF3FF',
     300: '#CBDEEB',
+    400: '#DFF3FF',
```

Comment:
Can we define color properly. Like give them a name or something. Ideally, this is used to add different shades to one color.

Reply:
Completed this